dashboard/src/components/InfoCard/InfoCard.v2.tsx

```diff
@@ -0,0 +1,84 @@
+import { ClarityIcons, infoCircleIcon } from "@clr/core/icon-shapes";
+import * as React from "react";
+import { Link } from "react-router-dom";
+import { CdsIcon } from "../Clarity/clarity";
+
+import placeholder from "../../placeholder.png";
+import "./InfoCard.v2.css";
+
+ClarityIcons.addIcons(infoCircleIcon);
+
+export interface IInfoCardProps {
+  title: string;
+  info: string | JSX.Element;
+  link?: string;
+  icon?: string;
+  banner?: string;
+  subIcon?: string;
+  description?: string | JSX.Element;
+  tag1Class?: string;
+  tag1Content?: string | JSX.Element;
+  tag2Class?: string;
+  tag2Content?: string | JSX.Element;
```

Comment:
Can we make these props more explicit? `tag1` and `tag2` may be confusing about the content they contain:

```
categoryClass:
category:
statusClass:
status:
```

Also, you can define a `StatusTag` and `CategoryTag` component that sets the colors automatically. In this way, the component that calls `InfoCard` doesn't need to know the CSS class for the tags.

Reply:
mmm still think it's not trivial to understand the purpose, but it's not a blocker for this PR :+1: